packages/xdl/src/Versions.js

```diff
@@ -3,16 +3,35 @@
  */
 
 import _ from 'lodash';
+import path from 'path';
 import semver from 'semver';
 
 import Api from './Api';
 import ApiV2Client from './ApiV2';
+import { Cacher } from './tools/FsCache';
 import ErrorCode from './ErrorCode';
 import XDLError from './XDLError';
 import UserManager from './User';
 
 export async function versionsAsync() {
-  return await Api.versionsAsync();
+  const api = new ApiV2Client();
+  const versionCache = new Cacher(
+    () => api.getAsync('versions/latest'),
```

Comment:
Can we use a URL that doesn’t do a redirect here? I previously had to switch from `/--/versions` to `/--/api/v2/versions` (which the former redirects to) after I discovered the redirect added a 1-2 second round trip to our servers (at least at the time). Because many commands call this API, so it should be as fast as possible.

Reply:
I think this won't redirect bc the ApiV2Client adds the `/--/api/v2` part for us: https://github.com/expo/expo-cli/blob/91239153591d989b37bef525c20dbaeb93573de8/packages/xdl/src/ApiV2.js#L23
I confirmed this by logging the url in the axios request object.

Reply:
Ah, no problem then. I was just confused because I saw the redirect here: https://github.com/expo/universe/pull/3431/files#diff-483720538a0595eb9edfa7d2d9ce5316R317.